frontend/cypress/integration/common/graph_display.ts

```diff
@@ -257,8 +264,9 @@ Then('user does not see service nodes', () => {
 Then('security {string} in the graph', action => {
   validateInput('filterSecurity', action);
 
-  cy.waitForReact(1000, '#root');
+  cy.waitForReact();
```

Comment:
Is the timeout param no longer needed?

Reply:
it was only needed because the second param was being provided.  Now the root definition has moved to the config.  The default timeout is 10s, I think it's fine but if you want me to lower it, I can.

Reply:
> The 1 failure in CI is unrelated to these changes and looks like a flake

Yes, that test succceded on some runs.  It is not failing due to these changes.